### PR TITLE
Fixed the setting of the various dir variables and usage of $BUILD_DIR

### PR DESCRIPTION
**Pull request by rspada-yesware**
Fixed the assignment of the various DIR variables - no spaces between the variable name and the equal signs.

Also fixed the usage of $BUILD_DIR

### CODE REVIEW

File: bin/compile [hunk 1]
```diff
@@ -12,21 +12,18 @@ ehco "$3 = " $3
...
-    echo "Empty BUILD_DIR.  Using $PWD"
-    BUILD_DIR = $PWD
-fi
-
+#
 # install the unixodbc driver first as it has some files that
 # are needed for snowflake odbc
+#
 echo 'Installing unix ODBC'
 curl http://www.unixodbc.org/unixODBC-2.3.4.tar.gz | tar xvz
```
Comment (mike-yesware) [inline bin/compile]: Is this okay, or should it be explicitly in a tmp location?

Comment (rspada-yesware) [inline bin/compile]: Yeah, you're right.  Putting this in tmp would be better.

**Comment by mike-yesware** [comment]
One question about an unrelated command, since directories are on my mind because of this PR.

**Comment by rspada-yesware** [comment]
Merging w/o a code review because we're still bootstrapping the buildpack.

**Pull request merged**